api/src/main/java/org/openmrs/module/xdssender/api/cda/model/DocumentModel.java

```diff
@@ -34,12 +36,15 @@
 	 * Can only be created by static method
 	 */
 	private DocumentModel() {
-		
 	}
 	
 	public ClinicalDocument getDoc() {
 		return doc;
 	}
+
+    public List<Author> getAuthors() {
+        return doc != null ? doc.getAuthor() : new ArrayList<Author>();
+    }
```

Comment:
There is something wrong with indentation. Tip: enable displaying white characters -> https://intellij-support.jetbrains.com/hc/en-us/community/posts/207045165-Show-non-printable-characters-Tabs-